pkg/arvo/app/chat-store.hoon

```diff
@@ -72,6 +81,74 @@
     ::
     ==
     ::
+    ++  nobody
+      ^-  @p
+      (bex 128)
```

Comment:
What made you decide on making a "null ship" vs discarding the data? Just curious, not strongly opinionated on the matter.

Reply:
I'm just incredibly hesitant to discard data, given the amount of data that's generally in chats

Reply:
I think this hesitation is generally correct — discarding user data is essentially forbidden as per previous discussions, so we should really be eager to instead dump malformed data and provide the means by which it can be amended, re-imported or just otherwise archived.